app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1474,8 +1489,19 @@ DeviceDelegateOpenXR::EnterVR(const crow::BrowserEGLContext& aEGLContext) {
     m.controllersReadyCallback = nullptr;
   }
 
-  // Initialize layers if needed
   vrb::RenderContextPtr context = m.context.lock();
+
+  // Lazily create hand mesh renderer
+  if (m.mHandTrackingSupported && !m.handMeshRenderer) {
+    auto& create = context->GetRenderThreadCreationContext();
+#if defined(PICOXR) || defined(SPACES)
```

Comment:
Now that I think about it, for SPACES we should probably render the hand model as the MSFT extension is supported, shouldn't we?

Reply:
Yes, I forgot to remove `SPACES` from the ifdef, will fix that.

Reply:
Actually, this series don't include support for MSFT extension yet, so here it is fine to have SPACES using the Spheres.

It will be removed as part of PR#821, which depends on this one.